.eslintrc.js

```diff
@@ -42,7 +42,9 @@ module.exports = {
         'one-var': ["error", "never"],
         'quotes': ["error", "single", { "avoidEscape": true, "allowTemplateLiterals": false }],
         'semi': ["error"],
-        'space-before-blocks': ["error"]
+        'space-before-blocks': ["error"],
+        // Disable while it's buggy
+        'compat/compat': ["off"]
```

Comment:
cannot we shut it up only in a place we know it's buggy? not blanket everywhere?

Reply:
I changed the way it works by adding the offending API to the polyfills temporarily. Had to disable one more line which wasn't caught before, for some reason.